tests/src/JIT/jit64/opt/cse/hugeexpr1.csproj

```diff
@@ -30,6 +30,9 @@
     <DebugType>Full</DebugType>
     <Optimize>True</Optimize>
   </PropertyGroup>
+  <PropertyGroup>
+    <JitOptimizationSensitive>true</JitOptimizationSensitive>
```

Comment:
You can make this `<JitOptimizationSensitive Condition="'$(Platform)' == 'x86'">true</JitOptimizationSensitive>` in order to disable this for x86 only.